interfaces/dotnet/Cantera.Tests/src/EnumsTest.cs

```diff
@@ -0,0 +1,67 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+namespace Cantera.Tests;
+
+public class EnumsTests
+{
+    [Fact]
+    public void ThermoPair_ToStringsCorrently()
+    {
+        var fields = typeof(ThermoPair)
+            .GetFields(BindingFlags.Public | BindingFlags.Static)
+            .Select(f =>
+            (
+                name: f.Name,
+                interopString: ((ThermoPair) f.GetValue(null)!).ToInteropString()
+            ));
+
+        var withShortNames = fields
+            .Where(f => f.name.Length == 2);
+
+        var withLongNames = fields
+            .Except(withShortNames);
+
+        // Ensure we have both long and short versions
+        Assert.Equal(withShortNames.Count(), withLongNames.Count());
+
+        foreach (var field in withShortNames)
+        {
+            Assert.Equal(field.name, field.interopString);
+
+            Assert.Single(withLongNames.Where(f => f.interopString == field.name));
+        }
+    }
+
+    [Fact]
+    public void EquilibriumSolver_MapsCorrectly() =>
+        TestInteropEnumInvariants<EquilibriumSolver>(true, -1, 2);
+
+    [Fact]
+    public void LogLevel_MapsCorrectly() =>
+        TestInteropEnumInvariants<LogLevel>(true, 0, 2);
+
+    static void TestInteropEnumInvariants<TEnum>(bool contiguous = false, int? min = null, int? max = null)
+        where TEnum : struct, Enum
+    {
+        Assert.Equal(typeof(int), Enum.GetUnderlyingType(typeof(TEnum)));
+
+        var values = Enum
+            .GetValues<TEnum>()
+            .Select(v => Unsafe.As<TEnum, int>(ref v))
+            .OrderBy(v => v)
+            .ToList();
+
+        if (contiguous) for (var i = 1; i < values.Count; i++)
+        {
```

Comment:
This is a personal preference: because C# style is that open braces _always_ begin on new lines, I sometimes elide them for small `if`s in order to keep things a bit smaller vertically, for example:
```
if (!myBool)
    return;

if (something) for (var x in y)
{
    ....
}

if (foo) using (var bar = new Bar)
{
    ...
}
```

Admittedly, this is a bit idiosyncratic, so let me know if you would prefer I just go with always using braces.

```
if (!myBool)
{
    return;
}

if (something)
{
    for (var x in y)
    {
        ....
    }
}

if (foo)
{
    using (var bar = new Bar)
    {
        ...
    }
}
```

(although note that in C# 9+ there you can write a `using` statement without needing braces, in which case the object will be in scope in the surrounding black)